src/main/java/com/revature/repositories/PersonRepository.java

```diff
@@ -10,7 +10,12 @@
 @Repository
 public interface PersonRepository extends JpaRepository<Person,Integer> {
 
-	List<Person> findByFirstName(String firstName);
+	Person getPersonById(int id);
+	
+//	List<Person> findAllPersons();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)